utilities/ideascale-importer/ideascale_importer/reviews_importer/processing/prepare.py

```diff
@@ -0,0 +1,82 @@
+"""Set of commands for the preparation of the review stage."""
+import typer
+import csv
+from typing import List
+
+from .tools.importer import Importer, IdeascaleImporter
+from .tools.allocator import Allocator
+from .tools.postprocessor import Postprocessor
+
+app = typer.Typer()
+
+
+async def allocate(
+    pas_path: str,
+    ideascale_api_key: str,
+    ideascale_api_url: str,
+    group_id: int, 
+    challenges_group_id: int,
+    stage_ids: List[int],
+    nr_allocations: List[int],
+    output_path: str,
+    anonymize_start_id: int = 5000,
+    seed: int = 7,
+):
+    """Run the allocator check."""
+
+    async def _allocate():
+        importer = Importer()
+        ideascale = IdeascaleImporter(ideascale_api_key, ideascale_api_url)
+        allocator = Allocator(importer, nr_allocations_map(), seed, ideascale)
+        importer.load_pas(pas_path)
+        await ideascale.import_challenges(challenges_group_id)
+        await ideascale.import_proposals(stage_ids=stage_ids)
+        await ideascale.import_com_revs(group_id=group_id, start_id=anonymize_start_id, historic_pas=importer.pas)
+        allocator.allocate()
+
+        # This data is not used by our core system, it is only consumed by Ideascale
+
+        # allocator.export_allocations(allocator.source.pas, f"{output_path}/allocations-by-pa.xlsx")
+        # groups = allocator.generate_challenges_groups()
+        # for group in groups:
+        #     allocator.export_allocations(group["pas"], f"{output_path}/{group['challenge']}.xlsx")
```

Comment:
Why is this commented out?

Reply:
Right now we are not exporting data in this format into the excel files, but probably we will want to automate this part also. But right now it is not in current scope, so that is why I left it as commented out